apps/js-machine-front/src/app/components/Snowflakes.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import './styles/snowflakes.css';
+import './styles/snowflakesMedia.css';
+
+import Switch from '@material-ui/core/Switch';
+import { makeStyles } from '@material-ui/core/styles';
+
+import { useSnowflakes } from './hooks/useSnowflakes';
+
+export function Snowflakes() {
+
+    const {isSnowing, handleSnowSwitch} = useSnowflakes();
+
+    const MAIN_YELLOW = '#F2E14C';
+
+    const useStyles = makeStyles({
```

Comment:
please create useStyles variable outside component, otherwise this variable will be create on each component Render